arm-monitor/2015-04-01/swagger/autoscale_API.json

```diff
@@ -1012,15 +1024,19 @@
           "type": "array",
           "items": {
             "type": "integer",
-            "format": "int32"
+            "format": "int32",
```

Comment:
I am removing the minimum/maximum from hours and minutes, the back-end rejects any value outside those limits anyway. It has always done that.